test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -811,14 +803,11 @@ func prepareForOnClusterLayeringTest(t *testing.T, cs *framework.ClientSet, test
 	})
 
 	imagestreamObjMeta := metav1.ObjectMeta{
-		Name:      "os-image",
-		Namespace: strings.ToLower(t.Name()),
+		Name: "os-image",
 	}
```

Comment:
This was necessary because we are no longer able to inject a custom pull secret. By placing the image in the MCO's slice of the registry, the daemon will be able to handle this rebase with existing secrets.